src/include/libpmemobj/action_base.h

```diff
@@ -72,9 +72,13 @@ struct pobj_action {
 };
 
 #define POBJ_MAX_ACTIONS 60
+#define POBJ_ACTION_XRESERVE_VALID_FLAGS\
+	POBJ_XALLOC_CLASS_MASK | POBJ_XALLOC_ZERO
```

Comment:
() around the value please.

Later you use this flag in this way: ~POBJ_ACTION_XRESERVE_VALID_FLAG